### ISSUE TITLE
[Feature Request] Import progress display

### ISSUE THREAD

**Issue by PeterDaveHello** [chewing/chewing-editor#194]
Currently, the import progress may take a long time for huge dictionary file, the UI [dialog to select file] will  hang on, but I can see the process is still working by enabling debug message, so maybe it'll be good to hide the dialog once the import process starting, and also have a progress bar or something to show to import progress.

**Comment by robert501128** [comment]
Is it possible to change the architecture of the importation progress? I think we definitely have to speed up the importation time if we want to import a huge dictionary in the future.

If so, where to start for changing the architecture?

**Comment by PeterDaveHello** [comment]
Yes we should speed it up, and there is no conflict with these two features, we can do them both.
